src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php

```diff
@@ -176,6 +177,46 @@ public function testFilterDuplicateTransPaths()
         $this->assertEquals($expectedPaths, $filteredTransPaths);
     }
 
+    /**
+     * @dataProvider removeNoFillProvider
+     */
+    public function testRemoveNoFillTranslationsMethod($noFillCounter, $messages)
+    {
+        // Preparing mock
+        $operation = $this->createMock(MessageCatalogueInterface::class);
+        $operation
+            ->method('all')
+            ->with('messages')
+            ->willReturn($messages);
+        $operation
+            ->expects($this->exactly($noFillCounter))
+            ->method('set');
+
+        // Calling private method
+        $translationUpdate = $this->createMock(TranslationUpdateCommand::class);
+        $reflection = new \ReflectionObject($translationUpdate);
+        $method = $reflection->getMethod('removeNoFillTranslations');
+        $method->setAccessible(true);
```

Comment:
Not needed anymore

See https://www.php.net/manual/en/reflectionmethod.setaccessible.php

Reply:
Done :)